pom.xml

```diff
@@ -4,16 +4,55 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 
-	<parent>
-		<groupId>tech.tablesaw</groupId>
-		<artifactId>tablesaw-parent</artifactId>
-		<version>0.38.2</version>
-	</parent>
-
+	<groupId>net.tlabs-data</groupId>
 	<artifactId>tablesaw-parquet</artifactId>
+	<version>0.38.2-SNAPSHOT</version>
```

Comment:
Don't use SNAPSHOT(s)

Reply:
Well.. that's the default behavior of the release plugin and of maven central... We will set non snapshot version numbers for actual releases.

Reply:
Versioning scheme will be defined later